test/benchmark/bench_linux_and_aster.sh

```diff
@@ -6,7 +6,12 @@ set -e
 set -o pipefail
 
 # Ensure all dependencies are installed
-command -v yq >/dev/null 2>&1 || command -v yq >/dev/null 2>&1 || { echo >&2 "tools are not installed. Aborting."; exit 1; }
+if ! command -v yq >/dev/null 2>&1 || ! command -v jq >/dev/null 2>&1; then
+    >&2 echo "Error: Missing required tools:"
+    ! command -v yq >/dev/null 2>&1 && >&2 echo "  - yq"
```

Comment:
If a single tool is missing, the error message would be

```
Error: Missing required tools:
   - yq
```

This is strange to me. I expect it to be

```
Error: missing required tools: yq
```

So I suggest rewriting the logic this way.

```
if ! command -v yq >/dev/null 2>&1; then
    echo >&2 "Error: missing required tool: yq"
    exit 1
fi
if ! command -v jq >/dev/null 2>&1; then
    echo >&2 "Error: missing required tool: jq"
    exit 1
fi
```